libs/sdk-ui-ext/src/internal/components/configurationControls/comparison/calculation/CalculationListItemInfo.tsx

```diff
@@ -0,0 +1,22 @@
+// (C) 2023 GoodData Corporation
+import React from "react";
+
+import CalculationListItemInfoSection from "./CalculationListItemInfoSection.js";
+
+interface ICalculationItemInfoProps {
+    title: string;
+    info: string;
```

Comment:
Is there a better name for this prop? I'm not sure what goes there unless I open the component source code and analyze what it does with this prop.

Reply:
Changed this property to calculationType!